### ISSUE TITLE
Does the VXR7200 need to be initialized?

### ISSUE THREAD

**Issue by waayst** [SimulaVR/VXR7200_Bridge#1]
Does the VXR7200 need to be initialized?

**Comment by KaneTW** [comment]
Yes. Unfortunately, the firmware generation software is under Synaptics NDA.

I can publish an implementation of the I2C configuration protocol, but it's not gonna help with the initial configuration.
